src/Confluent.Kafka/Impl/LibRdKafka.cs

```diff
@@ -1029,6 +1035,22 @@ internal static IntPtr consumer_group_metadata_read(out IntPtr cgmd, byte[] data
         private static Func<IntPtr, IntPtr> _memberid;
         internal static IntPtr memberid(IntPtr rk) => _memberid(rk);
 
+        private static Func<long, long, IntPtr> _uuid_new;
```

Comment:
These fields and methods should start with Uuid, as we just remove the rd_kafka prefix here.
```suggestion
        private static Func<long, long, IntPtr> _Uuid_new;
```